ocaml/xapi/message_forwarding.ml

```diff
@@ -1008,8 +1008,6 @@ functor
               debug "Pool.enable_tls_verification sleeping on fistpoint" ;
               Thread.delay 5.0
             done ;
-            Db.Cluster.get_all ~__context
-            |> List.iter (Xapi_cluster_helpers.Pem.maybe_write_new ~__context) ;
             all_hosts
             |> List.iter (fun host ->
                    do_op_on ~local_fn ~__context ~host (fun session_id rpc ->
```

Comment:
Since you loop over all hosts then this should be Client.Host.enable_tls_verification, not Client.Pool.enable_tls_verification (unrelated to clusterd changes, perhaps fix it in a separate commit).
If it was really a pool level operation then it should've taken a pool as an argument and do the looping internally (but you'd still need a host-level API to call).